ostd/src/mm/io.rs

```diff
@@ -393,6 +394,20 @@ pub struct VmReader<'a, Fallibility = Fallible> {
     phantom: PhantomData<(&'a [u8], Fallibility)>,
 }
 
+// `Clone` can be implemented for `VmReader`
+// because it points to either untyped memory or immutable references.
+// Note that we cannot implement `Clone` for `VmWriter`
+// because it can point to mutable references, which must be exclusive.
```

Comment:
Sorry for the noise. When I reread it, I found it a bit wired to say "point to ... references", maybe "... either points to untyped memory or represents immutable references"/"because it can represent mutable references, which ..." or something like that?